main.go

```diff
@@ -295,6 +296,8 @@ func main() {
 		influxdbTagsEnabled  = kingpin.Flag("statsd.parse-influxdb-tags", "Parse InfluxDB style tags. Enabled by default.").Default("true").Bool()
 		libratoTagsEnabled   = kingpin.Flag("statsd.parse-librato-tags", "Parse Librato style tags. Enabled by default.").Default("true").Bool()
 		signalFXTagsEnabled  = kingpin.Flag("statsd.parse-signalfx-tags", "Parse SignalFX style tags. Enabled by default.").Default("true").Bool()
+		relayHostPort        = kingpin.Flag("statsd.relay.hostport", "The UDP relay target host:port").String()
```

Comment:
```suggestion
		relayAddr        = kingpin.Flag("statsd.relay.address", "The UDP relay target address (host:port)").String()
```
to be consistent with `--web.listen-address` and the Go standard library.

Reply:
Ahh, I forgot to fix that.